content/docs/authorization/deployment/_index.md

```diff
@@ -200,29 +200,67 @@ There are currently 2 ways of doing this:
 
 Given a setup where Kubernetes, a storage system, CSI driver(s), and CSM for Authorization are deployed, follow the steps below to configure the CSI Drivers to work with the Authorization sidecar:
 
-Run the following commands on the CSI Driver host
+1. Create the secret token in the namespace of the driver.
 
-   ```console
-    # Specify Authorization host address. NOTE: this is not the same as IP
-    export AuthorizationHost=""
-
-    echo === Applying token token ===
-    # It is assumed that array type powermax has the namespace "powermax" and powerflex has the namepace "vxflexos"
+    ```console
+    # It is assumed that array type powermax has the namespace "powermax", powerflex has the namepace "vxflexos", and powerscale has the namespace "isilon".
     kubectl apply -f /tmp/token.yaml -n powermax
     kubectl apply -f /tmp/token.yaml -n vxflexos
-
-    echo === injecting sidecar in all CSI driver hosts that token has been applied to === 
-    sudo curl -k https://${AuthorizationHost}/install | sh
-    
-    # NOTE: you can also query parameters("namespace" and "proxy-port") with the curl url if you desire a specific behavior.
-    # 1) For instance, if you want to inject into just powermax, you can run
-    #    sudo curl -k https://${AuthorizationHost}/install?namespace=powermax | sh
-    # 2) If you want to specify the proxy-port for powermax to be 900001, you can run
-    #    sudo curl -k https://${AuthorizationHost}/install?proxy-port=powermax:900001 | sh
-    # 3) You can mix behaviors
-    #    sudo curl -k https://${AuthorizationHost}/install?namespace=powermax&proxy-port=powermax:900001&namespace=vxflexos | sh
+    kubectl apply -f /tmp/token.yaml -n isilon
    ```
 
+2. Edit the following parameters in samples/secret/karavi-authorization-config.json file in the CSI driver and update/add connection information for one or more backend storage arrays.
```

Comment:
Link to "samples/secret/karavi-authorization-config.json"

Reply:
This is a common deployment section. If we were to link, I think it would need to look like: "file in CSI PowerFlex, CSI PowerMax, or CSI PowerScale driver and ..."

Reply:
fixed